discard.go

```diff
@@ -0,0 +1,17 @@
+package astiav
+
+//#include <libavcodec/defs.h>
+import "C"
+
+// https://ffmpeg.org/doxygen/7.0/group__lavc__decoding.html#ga352363bce7d3ed82c101b3bc001d1c16
+type Discard C.enum_AVDiscard
+
+const (
+	DiscardNone          = Discard(C.AVDISCARD_NONE)     // discard nothing
```

Comment:
Can you remove all the comments on the right?

The link you've put above `type Discard C.enum_AVDiscard` is enough if the developer wants extra information  👍